Makefile

```diff
@@ -11,7 +11,9 @@ build:
 install:
 	$(PYTHON) setup.py install -O1 $(PYTHON_PREFIX_ARG) --root $(DESTDIR)
 	install -d $(DESTDIR)/etc/xdg/autostart
-	install -m 0644 etc/qvm-start-gui.desktop $(DESTDIR)/etc/xdg/autostart/
+	install -m 0644 etc/qvm-start-daemon.desktop $(DESTDIR)/etc/xdg/autostart/
+	install -d $(DESTDIR)/usr/bin
+	ln -sf /usr/bin/qubes-start-daemon $(DESTDIR)/usr/bin/qubes-start-gui
```

Comment:
1. qvm-start-gui and qvm-start-daemon
2. Use relative target path here (just `qvm-start-daemon`, not the full path)

Reply:
And also symlink man pages. And preferably add a note to the man page about legacy name.

Reply:
Yes thanks for pointing it out. I figured out this too when testing.